packages/sn-filter-pane/src/components/ListboxGrid/ListboxGrid.tsx

```diff
@@ -89,17 +89,21 @@ function ListboxGrid({ stores }: { stores: IStores }) {
   };
 
   const handleKeyDown = (event: React.KeyboardEvent) => {
+    const target = event.target as HTMLElement;
+    if (target.tagName === 'INPUT') {
+      return;
+    }
     if (event.key === KEYS.ESC && keyboard?.enabled) {
       // @ts-ignore
-      if (event.target.classList.contains('listbox-container')) {
+      if (target.classList.contains('listbox-container')) {
         // Focus currently on a listbox
         preventDefaultBehavior(event);
         // @ts-ignore
         keyboard.blur?.(true);
       }
     } else if (event.key === KEYS.LEFT || event.key === KEYS.RIGHT) {
       let elementToFocus;
-      const listboxList = gridRef?.current?.querySelectorAll && gridRef?.current?.querySelectorAll('.listbox-container');
+      const listboxList = gridRef?.current?.querySelectorAll && gridRef?.current?.querySelectorAll<HTMLElement>('.listbox-container');
```

Comment:
```suggestion
      const listboxList = gridRef?.current?.querySelectorAll?.<HTMLElement>('.listbox-container');
```

a little cleaner maybe

Reply:
that give a type error for me (I think it should not and maybe is a bug in typescript)